src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -162,10 +162,13 @@ private function set_element_value( DOMNode $element, $value ) {
 		if ( $element instanceof DOMAttr ) {
 			$element->parentNode->setAttribute( $element->name, $value );
 		} else {
-			$fragment = $element->ownerDocument->createDocumentFragment();
-			$fragment->appendXML( $value );
-			$clone = $element->cloneNode(); // Get element copy without children
-			$clone->appendChild( $fragment );
+			$clone = $element->cloneNode( false ); // Get element copy without children
```

Comment:
@BruceOnTheGo I either need to read what `false` means or rely in your comment.

Wouldn't be better to create a `cloneNodeWithoutChildren()` method which works as a wrapper of `$element->cloneNode( false )`?

That way you don't need to add any comment here.

Reply:
Done